flake/dev/flake.nix

```
@@ -4,32 +4,127 @@
+  #               };
+  #             };
+  #
+  #             # keep-sorted start block=yes newline_separated=yes
+  #             flake-compat.url = "github:edolstra/flake-compat";
+  #
+  #             git-hooks = {
+  #               url = "github:cachix/git-hooks.nix";
+  #               inputs = {
+  #                 flake-compat.follows = "flake-compat";
+  #                 nixpkgs.follows = "stylix/nixpkgs";
+  #               };
+  #             };
+  #
+  #             home-manager = {
+  #               url = "github:nix-community/home-manager";
+  #               inputs.nixpkgs.follows = "stylix/nixpkgs";
+  #             };
+  #
+  #             nixvim = {
+  #               url = "github:nix-community/nixvim";
+  #               inputs = {
+  #                 flake-parts.follows = "stylix/flake-parts";
+  #                 nixpkgs.follows = "stylix/nixpkgs";
+  #                 systems.follows = "stylix/systems";
+  #               };
+  #             };
+  #
+  #             nvf = {
+  #               url = "github:NotAShelf/nvf";
+  #               inputs = {
+  #                 nixpkgs.follows = "stylix/nixpkgs";
+  #                 systems.follows = "stylix/systems";
+  #               };
+  #             };
+  #             # keep-sorted end
+  #           };
+  #
+  #       once 26.05 is released, giving non-flake end-users and Stylix
+  #       contributors two LTS releases to adopt Nix 2.26+ [1]. Note that
+  #       non-nixpkgs public flake inputs should be disabled.
+  #
+  #       [1]: https://github.com/NixOS/nix/blob/d4f67fd46dfe2bc950bdfa14273f87b8a4c32e47/doc/manual/source/release-notes/rl-2.26.md?plain=1#L3-L11
   inputs = {
-    # We need to define a nixpkgs input so that other inputs can follow it.
-    # It is prefixed with 'dev-' to avoid shadowing the public flake's
-    # 'nixpkgs' input, which could affect local testing with --override-input.
-    #
-    # TODO: Replace
-    #
-    #           dev-nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
-    #
-    #       with
-    #
-    #           dev-nixpkgs = {
-    #             url = "path:../..";
-    #             inputs = {
-    #               "<PUBLIC_INPUT_1>".follows = "";
-    #               "<PUBLIC_INPUT_2>".follows = "";
-    #               "<PUBLIC_INPUT_3>".follows = "";
-    #             };
-    #           };
-    #
-    #       once 26.05 is released, giving non-flake end-users and Stylix
-    #       contributors two LTS releases to adopt Nix 2.26+ [1]. Note that
-    #       non-nixpkgs public flake inputs should be disabled.
-    #
-    #       [1]: https://github.com/NixOS/nix/blob/d4f67fd46dfe2bc950bdfa14273f87b8a4c32e47/doc/manual/source/release-notes/rl-2.26.md?plain=1#L3-L11
+    dev-flake-parts = {
+      url = "github:hercules-ci/flake-parts";
+      inputs.nixpkgs-lib.follows = "dev-nixpkgs";
+    };
+
     dev-nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+    dev-systems.url = "github:nix-systems/default";
```

to match formatting in `/flake.nix`
```suggestion
    dev-nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
    
    dev-flake-parts = {
      url = "github:hercules-ci/flake-parts";
      inputs.nixpkgs-lib.follows = "dev-nixpkgs";
    };
    
    dev-systems.url = "github:nix-systems/default";
```